# The "encoding" is no more supported in python json library (utf-8 is default now?)
# author: Jiri Furst <jiri.furst@gmail.com>

--- qtwebkit-5.212.0-alpha4/Source/JavaScriptCore/generate-bytecode-files.orig	2022-12-20 15:43:51.483563960 +0100
+++ qtwebkit-5.212.0-alpha4/Source/JavaScriptCore/generate-bytecode-files	2022-12-20 15:43:23.979251951 +0100
@@ -163,7 +163,8 @@
         initBytecodesFile = openOrExit(initASMFileName, "w")
 
     try:
-        bytecodeSections = json.load(bytecodeFile, encoding = "utf-8")
+        #bytecodeSections = json.load(bytecodeFile, encoding = "utf-8")
+        bytecodeSections = json.load(bytecodeFile)
     except:
         print("Unexpected error parsing {0}: {1}".format(bytecodeJSONFile, sys.exc_info()))
 
